README.md

```diff
@@ -15,6 +15,30 @@ vendor/bin/drush pm:enable os2forms_fasit
 Configure Fasit API `base url` and a way of getting
 certificate on `/admin/os2forms_fasit/settings`.
 
+### Certificate
+
+The certificate must be in `pem` or `cer` format and
+must be whitelisted by Fasit Schultz.
+For this the certificate thumbprint is needed. To get the thumbprint
+from the command line run
+
+```sh
+openssl x509 -in SOME_CERTIFICATE.pem -noout -fingerprint
+```
+
+Example output
+
+```sh
+SHA1 Fingerprint=6A:CB:26:1F:39:31:72:D8:7F:A3:99:7C:EC:86:56:97:59:A8:52:8A
+```
+
+which should be converted to lowercase and have its colons removed,
```

Comment:
The command line is your friend:

```shell
openssl x509 -in SOME_CERTIFICATE.pem -noout -fingerprint |  cut -d= -f2 | sed 's/://g' | tr '[:upper:]' '[:lower:]'
```

(that's just one way to do it)

Reply:
Updated with command.